### PR TITLE
[TIMOB-23509] iOS10: Support new iOS 10 blurring effects

### PR DESCRIPTION
**Pull request by hansemannn**
**JIRA:** https://jira.appcelerator.org/browse/TIMOB-23509

### CODE REVIEW

File: apidoc/Titanium/UI/iOS/iOS.yml [hunk 1]
```diff
@@ -917,6 +917,26 @@ properties:
     since: "5.4.0"
     osver: {ios: {min: "8.0"}}
 
+  - name: BLUR_EFFECT_STYLE_DEFAULT
```
Comment (cheekiatng) [inline apidoc/Titanium/UI/iOS/iOS.yml]: it's REGULAR. not DEFAULT.

File: apidoc/Titanium/UI/iOS/iOS.yml [hunk 2]
```diff
@@ -917,6 +917,26 @@ properties:
...
+    summary: |
+        Use with [BlurView.effect](Titanium.UI.iOS.BlurView.effect) to specify a blur effect.
+    description: |
+        Creates a blurring effect in the view. The blurring effect is the default one specified in iOS 10.
```
Comment (cheekiatng) [inline apidoc/Titanium/UI/iOS/iOS.yml]: same here. regular not default.

**Comment by cheekiatng** [comment]
FT passed. @hansemannn please address docs error. 
FYI the visual effect of regular = light, and prominent = extra light. I guess Apple decided to have better naming conventions for these blur effects.

File: apidoc/Titanium/UI/iOS/iOS.yml (hunk 2 above)
Comment (hansemannn) [inline apidoc/Titanium/UI/iOS/iOS.yml]: Isn't that the same line?

**Comment by hansemannn** [comment]
Updated docs, left a note for the iOS 10 naming.

File: apidoc/Titanium/UI/iOS/iOS.yml (hunk 2 above)
Comment (cheekiatng) [inline apidoc/Titanium/UI/iOS/iOS.yml]: The blurring effect is the `default` one specified in iOS 10.
vs
The blurring effect is the `regular` one specified in iOS 10.

**Comment by cheekiatng** [comment]
APPROVED

**Pull request merged**